elements/link-modal.js

```diff
@@ -120,12 +120,18 @@ function createModal () {
   let link = ''
   let el = ''
 
-  const modal = Modal({ onunload, onexit, render })
+  const modal = Modal({
+    onunload,
+    onexit,
+    render,
+    class: 'modal'
+  })
 
   return modal
 
   function onexit () {
```

Comment:
Not a fan of using `this` in any form external facing API; feel it's best left for internals